src/modules/vendor/pages/ni/management/CoursesDirectory.vue

```diff
@@ -95,6 +106,10 @@ export default {
     async refreshCourses () {
       try {
         this.courses = await Courses.list();
+        this.coursesWithGroupedSlot = await this.courses.map(course => ({
+          ...course,
+          slots: Object.values(groupBy(course.slots, s => this.$moment(s.startDate).format('DD/MM/YYYY'))),
+        }));
```

Comment:
Tu n'as pas besoin du `await` devant le `map` et `coursesWithGroupedSlot ` pourrait être une computed, qu'en penses-tu ?

Reply:
en fait ici je me demande si `this.course` est vraiment utile  maintenant qu'on a `coursesWithGroupedSlot `?

Reply:
oui tu as raison, on pourrait juste utiliser une variable normale et ainsi éviter de rendre la liste de formation réactive.